packages/bff/src/plugins/userApi.ts

```diff
@@ -0,0 +1,62 @@
+import { FastifyPluginAsync, FastifyReply, FastifyRequest } from 'fastify';
+import fp from 'fastify-plugin';
+import { ProfileRepository } from '../db';
+import { Profile } from '../entities/Profile';
+
+const getOrCreateProfile = async (sub: string, locale: string): Promise<Profile> => {
```

Comment:
Since we're moving stuff around:

- I think that the `getOrCreateProfile`-function should live in the `./src/entities/Profile.ts`-file.